assets/src/application/ui/display/icons/svgs/check.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+	aria-hidden="true"
+	class="ee-svg"
+	fill="currentColor"
+	focusable="false"
+	viewBox="64 64 896 896"
+	data-icon="check-circle"
+	width="1em"
+	height="1em"
```

Comment:
these need to either be set to `20` or `1.25rem`
`1em === 16px` which is too small
can you fix this for all of the icons plz?

Reply:
done

Reply:
it would be nice to have a styleguide or a story book, where we can check icons and other components. I notice that some icons have different default size, might be because of the viewbox and it might need some manual adjustments to make it look the same width as others. One example is `InfoCircleOutlined`, but this is how antd have exposed that

Reply:
AFAIK viewbox is the size of the grid used to create the SVG and is independent from the height and width of the rendered result.

so one SVG might have been created using a 100 x 100 viewbox, and another with a 10 x 10 viewbox, but can both be displayed at 20 x 20 where the first would be scaled down to 1/5 its size and the other would be scaled up to twice its size

Reply:
yes, agree with that;
will need to test that exact svg separately to see why the sizings are a little bit different